sdk/program/src/stake/instruction.rs

```diff
@@ -229,7 +229,12 @@ pub enum StakeInstruction {
     ///   None
     ///
     /// The minimum delegation will be returned via the transaction context's returndata.
-    /// Use `get_return_data()` to retrieve the result.
+    /// Use [`get_return_data()`] to retrieve the result.  Alternatively, use the
+    /// [`get_minimum_delegation_return_data()`] or [`get_minimum_delegation()`] helper functions.
+    ///
+    /// [`get_return_data()`]: crate::program::get_return_data
+    /// [`get_minimum_delegation_return_data()`]: super::tools::get_minimum_delegation_return_data
+    /// [`get_minimum_delegation()`]: super::tools::get_minimum_delegation
```

Comment:
I would prefer if make it clear that the `stake::tools::get_minimum_delegation` function is basically always what program developers should use.

Reply:
Will do.